api/settings/base.py

```diff
@@ -148,9 +150,22 @@
 
 USE_TZ = True
 
+# Media files
+
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media_files')
+MEDIA_URL = '/media/'
+
+
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/dev/howto/static-files/
 
+STATIC_ROOT = os.path.join(BASE_DIR, 'allstatic')
 STATIC_URL = '/static/'
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, 'static'),
```

Comment:
Its wrong practice use `os.path.join` in this settings. Please check docs and use simple path `STATIC_ROOT = /allstatic/` If folder put in BASE_DIR you can use simple path without `os.path.join`.